includes/utils/class-amp-dom-utils.php

```diff
@@ -22,6 +22,7 @@ class AMP_DOM_Utils {
 	 *
 	 * @since 1.2.1
 	 * @deprecated Use AmpProject\Dom\Document::AMP_BIND_DATA_ATTR_PREFIX instead.
+	 * @internal
```

Comment:
Is this and the subsequent instances actually needed since the entire class is `@internal`?